constructor-manager-ui/src/constructor_manager_ui/main.py

```diff
@@ -21,6 +441,53 @@ def main(package_name):
         Name of the package that the installation manager is handling.
     """
     app = QApplication([])
-    dialog = Dialog()
-    dialog.setWindowTitle(f"{package_name} installation manager")
+    app.setStyleSheet(open(QSS_STYLESHEET, "r").read())
```

Comment:
Could we move this logic to a separate method and have a context manager for the file?

Also, since we are using a bunch of colors in different places. Could we unify them and use a simple replace mechanism like explained here https://stackoverflow.com/a/11031994 or even use https://github.com/spyder-ide/qtsass ?

```python
def _load_styles(self, variables{}):
    data = f.read()
    # do some qt-sass or simple variable replacement that could be overriden if variables is passed
    return parsed_data

def update_styles(self, variables={}):
    with open(QSS_STYLESHEET, "r") as f:
        app.setStyleSheet(self._load_styles(variables))
```

It can help in debugging and making updates to the palette and the styles :)